chain/messagepool/messagepool.go

```diff
@@ -431,10 +491,6 @@ func (mp *MessagePool) addLocked(m *types.SignedMessage) error {
 		mp.blsSigCache.Add(m.Cid(), m.Signature)
 	}
 
-	if m.Message.GasLimit > build.BlockGasLimit {
-		return xerrors.Errorf("given message has too high of a gas limit")
-	}
-
```

Comment:
this is checked by `ValidForBlockInclusion`, so it was redundant here and I removed it.